src/styles/blocks/menu.scss

```diff
@@ -0,0 +1,87 @@
+.menu {
```

Comment:
the menu should have a proper height (100% of the screen), pay attention to mobile and tablet versions as well :arrow_down: 

![image](https://user-images.githubusercontent.com/86923089/191077344-2ec32dd3-ab70-44ce-a277-2e3707dfee77.png)